tests/FsUnit.MsTest.Test/beGreaterThanTests.fs

```diff
@@ -1,26 +1,33 @@
 ﻿namespace FsUnit.Test
 open Microsoft.VisualStudio.TestTools.UnitTesting
 open FsUnit.MsTest
-open NHamcrest.Core
 
 [<TestClass>]
 type ``be greaterThan tests`` ()=
+
     [<TestMethod>] member test.
      ``11 should be greater than 10`` ()=
         11 |> should be (greaterThan 10)
 
     [<TestMethod>] member test.
-     ``11.1 should be greater than 11.0`` ()=
+     ``11,1 should be greater than 11,0`` ()=
```

Comment:
Why you replaced `.` in test names by `,`?

Reply:
This tests with `.` doesn't show up in my test explorer. Should I undo this?

Reply:
in VS?

Reply:
I thought it already fixed https://github.com/microsoft/testfx/issues/682

Reply:
Yes, in VS 16.6.2.

Reply:
I discovered that tests with period in their name are shown in a separate section within the test explorer.
In the coming PR I will undo this, because they're still there. I'm not satisfied with the `,` or other sequences (e.g. `..`) inside the test name.

Reply:
@sergey-tihon FYI: https://github.com/dotnet/fsharp/issues/9609